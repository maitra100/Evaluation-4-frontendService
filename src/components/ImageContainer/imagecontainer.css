#loginLeft{
    height:100vh;
    width:100%;
    display:flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    background-color: #f3f5ff;
}

#frontimage{
    width:500px;
    height:400px;
}

#text1{
    width: 100%;
    font-size: 40px;
    padding-left: 30%;
}

#text2{
    width: 100%;
    font-size: 40px;
    padding-left: 30%;
}
