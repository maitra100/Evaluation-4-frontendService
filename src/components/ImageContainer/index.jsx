import React from 'react';
import './imagecontainer.css';
import frontpic from '../../assets/Images/frontpic.png';

function ImageContainer() {
  return (
    <div id="loginLeft">
      <div id="text1">Design APIs fast,</div>
      <div id="text2">Manage content easily</div>
      <img id="frontimage" src={frontpic} alt="frontpic" />
    </div>
  );
}

export default ImageContainer;
