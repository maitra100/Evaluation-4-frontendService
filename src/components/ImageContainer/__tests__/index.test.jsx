import renderer from 'react-test-renderer';
import React from 'react';
import ImageContainer from '../index';

jest.mock('react-router-dom');

describe('Login Section', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ImageContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
