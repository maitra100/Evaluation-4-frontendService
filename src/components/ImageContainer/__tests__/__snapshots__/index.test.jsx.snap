// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Section renders correctly 1`] = `
<div
  id="loginLeft"
>
  <div
    id="text1"
  >
    Design APIs fast,
  </div>
  <div
    id="text2"
  >
    Manage content easily
  </div>
  <img
    alt="frontpic"
    id="frontimage"
    src="frontpic.png"
  />
</div>
`;
